framework/doc/content/source/postprocessors/SolutionChangeOverTimePostprocessor.md

```diff
@@ -0,0 +1,25 @@
+# SolutionChangeOverTimePostprocessor
+
+The SolutionChangeOverTimePostprocessor returns the relative differential
+norm of the solution between the two previous time steps. For example, if this postprocessor
+is called on time step 3, it will display the relative change between time step 2 and
```

Comment:
```suggestion
is called on the beginning of time step 3, it will display the relative change between time step 2 and
```

I would assume it does 3 & 2 at the end of 3?

Reply:
No, it does not unfortunately. Because the point at which the steady-state differential norm is computed (and stored in `_solution_change_norm`) happens AFTER `EXEC_FINAL`. I couldn't figure out how to not have it be lagged

Reply:
Would you prefer that?

Reply:
Yes, as mentioned in the original post, I'd prefer to not have that time lag. But I am not sure how to go about it.

Reply:
My first guess is that there'd need to be another exec flag enum which is, like, `ACTUAL_EXEC_FINAL` which occurs after the steady-state norm is evaluated.

Reply:
I'd rather not have another exec flag. I ll have a look.

Reply:
Haha same, thanks so much